Dockerfile.QA

```diff
@@ -149,8 +149,8 @@ RUN mkdir -p qa/common && \
     cp bin/triton_json_test qa/L0_json/. && \
     cp bin/backend_output_detail_test qa/L0_backend_output_detail/. && \
     cp -r deploy/mlflow-triton-plugin qa/L0_mlflow/. && \
-    cp bin/input_byte_size_test qa/L0_input_validation/. && \
-    cp -r docs/examples/model_repository/simple_identity qa/L0_input_validation/models
+    cp -r docs/examples/model_repository/{simple,simple_identity,simple_string} qa/L0_input_validation/models && \
```

Comment:
Do we have any documentation surrounding this byte size validation? It seems like there's a list of exceptions, edge cases, etc. for different input types, different memory types, different backend types (python backend had an issue if I recall correctly?), core vs client vs backend checks, etc. -- it would be good to have it written down somewhere on the expected behaviors and known issues. 

Not sure this needs to be a public-facing doc, but would be valuable for us to have a reference even if only internal.

Reply:
Where is be the best place to document?

Reply:
Spoke offline